pkg/volume/util/atomic_writer_test.go

```diff
@@ -29,29 +29,17 @@ import (
 	"testing"
 
 	"k8s.io/apimachinery/pkg/util/sets"
-	utiltesting "k8s.io/client-go/util/testing"
 )
 
 func TestNewAtomicWriter(t *testing.T) {
-	targetDir, err := utiltesting.MkTmpdir("atomic-write")
-	if err != nil {
-		t.Fatalf("unexpected error creating tmp dir: %v", err)
-	}
-	defer os.RemoveAll(targetDir)
+	targetDir := t.TempDir()
 
 	_, err = NewAtomicWriter(targetDir, "-test-")
 	if err != nil {
 		t.Fatalf("unexpected error creating writer for existing target dir: %v", err)
 	}
 
-	nonExistentDir, err := utiltesting.MkTmpdir("atomic-write")
-	if err != nil {
-		t.Fatalf("unexpected error creating tmp dir: %v", err)
-	}
-	err = os.Remove(nonExistentDir)
-	if err != nil {
-		t.Fatalf("unexpected error ensuring dir %v does not exist: %v", nonExistentDir, err)
-	}
+	nonExistentDir := t.TempDir()
```

Comment:
It's not non-existent directory anymore, so we probably want to keep original code.

Reply:
Did you miss above comment?